src/main/java/uk/gov/hmcts/reform/sandl/snlrules/model/BookableJudge.java

```diff
@@ -35,7 +35,8 @@ public OffsetDateTime getEnd() {
 
     @Override
     public String toDescription() {
-        return ("Start: " + DateTimeUtils.humanizeDate(start) + ", duration: " + duration)
+        String readableDuration = DateTimeUtils.readableDuration(this.duration);
```

Comment:
I do not feel like this variable is needed? why just not to use this function inline.. hard to read IMHO (the smae for other places where it is written the same way